h/security/identity.py

```diff
@@ -84,6 +84,14 @@ class Identity:
     user: Optional[LongLivedUser] = None
     auth_client: Optional[LongLivedAuthClient] = None
 
+    @property
+    def is_lms(self):
```

Comment:
This I found while doing looking at security. I reckon `is_lms` is going to be useful right now.

Of course this won't be in a PR about adding a new column.